cid/common.py

```diff
@@ -1231,15 +1322,49 @@ def create_or_update_view(self, view_name: str, recursive: bool=True, update: bo
                     self.glue.create_or_update_table(view_name, view_query)
                 else:
                     if 'CREATE OR REPLACE' in view_query.upper():
-                        print(f'Updating view: "{view_name}"')
-                        self.athena.execute_query(view_query)
+                        update_view = False
+                        while get_parameters().get('on-drift', 'show').lower() != 'override' and isatty():
+                            cid_print(f'Analysing view {view_name}')
+                            diff = self.athena.get_view_diff(view_name, view_query)
+                            if diff and diff['diff']:
```

Comment:
same comment, could it be in the helper?

Reply:
same answer as above :)